# DepthSensorState.h

float64 depth